ParisBNF/abb/BNFabb3.xml

```diff
@@ -0,0 +1,327 @@
+                        <title type="complete" ref="LIT2509Weddas"/>
+                      </msItem>
+                     </msItem>
+               </msContents>
+
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <watermark/>
+                           <material key="parchment"/>
+
+                        </support>
+                        <extent>
+                           <measure unit="leaf">129</measure>
+                           <measure unit="leaf" type="blank">1</measure>
+                           <locus target="#105r"/>
+
+                           <dimensions type="outer" unit="mm">
+                              <height>198</height>
+                              <width>123</width>
+                           </dimensions>
+                        </extent>
+                        <foliation/>
+                        <collation>
+                        </collation>
+                        <condition key="good"/>
+                     </supportDesc>
+                     <layoutDesc>
+                        <layout columns="1" writtenLines="26">
+                           <locus from="1r" to="114v"/>
+                        </layout>
+                        <layout columns="2" writtenLines="26">
+                           <locus from="115v" to="128v"/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script">18th-century script</seg>
+                        <date notBefore="1700" notAfter="1799"/>
+                        <seg type="ink">Black, red</seg>
+                        </handNote>
+                  </handDesc>
+
+                  <additions>
+                     <list>
+                        <item xml:id="a1">
+                           <locus target="#128r"/>
+                           <desc type="Unclear">
+                             A note related to tributes,
+                             possibly in the region of <placeName ref="LOC1572Asgade"/>.</desc>
+                          <q xml:lang="am">
+                            ከ<placeName ref="LOC1572Asgade">አስገዴ፡</placeName> ፭፻፵፱ለሸለቃ፡ ሽቴ፡ ፫፻፺፭ለአዕበዮ፡ አራ፡
+                            ፬፻፸፩ለገብራይ፡ አድአቀይት፡ ፬፻፹፫ለአሰላፊ፡ ወልደ፡ ሚካኤል፡ <sic resp="DN">፫፻፺፮ለወድ፡</sic> ሸለቃ፡
+                             ፍሥሐ፡ ፫፻፷፪ለተስፋይ፡ ግባ፡ ደግ፡ <sic resp="DN">፬፻ለወድ፡</sic> ሸለቃ፡ ፍሥሐ፡ ገብረ፡ ሚካኤል።
+                              </q>
+                              <note>French translation in <bibl><ptr target="bm:ContiRossini1914Notice"/>
+                            <citedRange unit="page">49</citedRange></bibl>; words <foreign>waddi</foreign>
+                          and <foreign>ʾasallāfi</foreign> are noted as Tǝgrǝñña forms.</note>
```

Comment:
I understand that you provide additional information from the literature, but the way it is encoded now will not allow people who filter by Tigrinya to find this manuscript.

Reply:
It is said in the catalogue. Should I simply marke them as Tigrigna words?

Reply:
Yes, I think it would be better, in addition to the remarks from the catalogue

Reply:
I marked them up, but language identification dislikes <language ident="tn">Tǝgrǝñña</language>, does it (I left "ti" for the moment)?

Reply:
does the schema complain against `<foreign xml:lang="ti">`?